db/seeds.rb

```diff
@@ -148,6 +148,23 @@ def update_or_new(klass, attributes)
   r.feed_format = "video"
 end
 
+# Add random events
+
+update_or_new(Event, external_id: "Tucker's final helios PR :(") do |r|
+  r.source = 'github_pull'
+  r.created_at = Date.parse('2021-10-6T10:09:08.07')
```

Comment:
These dates are hard coded, what are the implications of that?

Reply:
The 'MojoTech by the Numbers' widget on helios displays events only for the week (I believe). I reached out to Will before he left for PTO to ask what he had in-mind for this half of the PR... These sample events are to be added as a display test for helios, and only appear on the widget for a temporary amount of time.